js/main.js

```diff
@@ -1,42 +1,157 @@
+  'Midtown Luxury Apartments by Sweet Inn',
+  'Room Mate Carla',
+  'Suite Home Sagrada Familia',
+  'Hotel Paxton Barcelona',
+  'Hotel Soho',
+  'Castro Exclusive Residences Sant Pau',
+  'Hostal Orleans',
+];
 
-  return randomNumber;
-}
+const TYPES = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+
+const TIMES = ['12:00', '13:00', '14:00'];
+
+const FEATURES = ['wifi', 'dishwasher','parking','washer', 'elevator', 'conditioner'];
+
+const DESCRIPTIONS = [
+  'До пляжа можно дойти всего за 1 минуту. Построенный в виде паруса восхитительный отель расположен рядом с пляжем Барселонета. Гости по достоинству оценят потрясающий дизайн отеля и фантастический вид на Барселону. К услугам гостей спа-центр, пейзажный бассейн, бар на крыше и роскошные номера.',
+  'В оформлении номеров использованы красочные принты. В числе удобств кондиционер, телевизор с плоским экраном, мини-бар, сейф и собственная ванная комната.',
+  'Апартаменты расположены на зеленом проспекте Гауди, всего в 250 метрах от собора Святого Семейства. Из окон некоторых апартаментов открывается вид на знаменитую достопримечательность Барселоны. Все апартаменты полностью оборудованы и оснащены кондиционером. Предоставляется бесплатный Wi-Fi.',
+  'Стильный отель расположен на улице Гран-Виа в центре Барселоны. К услугам гостей сезонный бассейн на крыше и терраса для загара с панорамным видом. Прогулка от отеля до площади Каталонии и бульвара Пасео-де-Грасиа занимает 10 минут.',
+  'В распоряжении гостей номера с рабочим столом, кондиционером, телевизором с плоским экраном, сейфом и собственной ванной комнатой с душем. Гостям предоставляются полотенца и постельное белье.',
+];
+
+const PHOTOLINKS = [
+  'duonguyen-8LrGtIxxa4w.jpg',
+  'brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const QUANTITYADS = 10;
+// function random number from two integers number
+
+const getRandomNumber = (a, b) => {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+
+// function random number with floating points from two integers numbers
+
+const getFloatingPointNumber = (a, b, digits = 1) => {
+  const lower = Math.min(Math.abs(a), Math.abs(b));
+  const upper = Math.max(Math.abs(a), Math.abs(b));
+  const result = Math.random() * (upper - lower) + lower;
+  return Number(result.toFixed(digits));
+};
+
+// get random avatar number
 
-getRandomNumber(0,3);
+const getNumberAvatar = () => {
+  const numberPhoto = String(getRandomNumber(1, 10));
+  return (numberPhoto.length === 1) ? `0${numberPhoto}` : numberPhoto;
+};
 
+// get author array
 
-// function get random number with floating points from two integers number
-// and number of digits after the decimal point
+const getAuthor = () => {
+  const authorArr = {};
+  authorArr['avatar'] = `img/avatars/user${getNumberAvatar()}.png`;
```

Comment:
```suggestion
  const authorArr = {
    avatar: `img/avatars/user${getNumberAvatar()}.png`
  };
```
лучше так. а ещё лучше, создавай этот объект в `outAdd` функции. чтобы все было консистентно